Cargo.toml

```diff
@@ -21,6 +21,9 @@ members = [
   "xtask",
 ]
 
+[patch.crates-io]
```

Comment:
Before merge we need to remove this=) Created PR with release https://github.com/FuelLabs/fuel-vm/pull/239

Reply:
You can use new release now=) https://github.com/FuelLabs/fuel-vm/releases/tag/v0.19.0